frontend/src/types/CamperTypes.ts

```diff
@@ -51,7 +51,7 @@ export type CreateCamperDTO = Omit<
 };
 
 export type EditCamperInfoFields = Omit<
-  Camper,
+  Partial<Camper>,
```

Comment:
Can you explain why this was needed? Don't fully understand

Reply:
Yep! Firstly, from my POV, `EditCamperInfoFields` is used to store fields to update for camper(s). For example, filling in `EditCamperInfoField.hasPaid` means to say that. "hey, we want these camper(s) to have their hasPaid set to whatever". Previously, `EditCamperInfoFields` was forced to have some fields (exp, firstname and lastname) be required. I wanted to use the `EditCamperInfoFields` type for moving camper(s) by setting `EditCamperInfoFields.sessionId`. In order to do so, I needed all of those required fields to be optional.

Reply:
Ah okay that makes sense, it's good that you're not making new types. My only concern is that by making camper Partial, basically everything else we've defined is optional too (maybe dangerous?), but seeing as we're nearing the end of the project, we should be alright.